modules/aws_account_permissions/variables.tf

```
@@ -0,0 +1,32 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# Required PARAMETERS
+# These variables have default values and can be modified based on the requirement.
+# ---------------------------------------------------------------------------------------------------------------------
+
+
+variable "iam_role_name" {
+  description = "Customize the name of IAM role for Illumio AWS integration"
+  type        = string
+  default     = "IllumioCloudIntegrationRole"
+  validation {
+    condition     = length(var.iam_role_name) > 1
+    error_message = "The IAM role name cannot be empty."
+  }
+}
+
+variable "access_mode" {
+  description = "integration mode Read/ReadWrite."
```

1. Don't refer to "integrations". The Terraform schema doesn't have any such concept.
2. Just copy the description from the Terraform schema as-is, to be consistent. https://github.com/illumio/terraform-provider-illumio-cloudsecure/blob/cd49f36a7d31ff3e3d339fc63846789c099e573e/api/schema/aws_account_resource.go#L37C21-L37C82
```suggestion
  description = "The account's access mode, must be `\"ReadWrite\"` (default) or `\"Read\"`."
```

---

I fixed this.